azimuth_caas_operator/tests/models/test_crds.py

```diff
@@ -58,6 +58,10 @@ def test_cluster_type_crd_json(self):
                   "playbook": {
                     "type": "string"
                   },
+                  "jobTimeout": {
+                    "nullable": true,
```

Comment:
The change to the model would mean this is not nullable, which I think is correct.

Reply:
I thought it meant null gets parsed in as 1200?

Reply:
No.

Reply:
The default is used if no value is provided at all. `schema.Optional[int]` means that the end user could specify `jobTimeout: null` in their CRD spec and that would be valid, which is not what we want.

Reply:
So there is a difference between nullable and required I guess?

Reply:
Yes. Nullable declares null as a valid value. A field can be required and nullable, which forces you to specify null for the null case.